### ISSUE TITLE
did tflearn support early stopping mechanism?

### ISSUE THREAD

**Issue by mlinxiang** [tflearn/tflearn#184]
when i do one experiment on nlp, the training process display as follow, which i think it assume to be overfiting atfer same epochs. i think is there has mechanism to fix this training problem
![image](https://cloud.githubusercontent.com/assets/6283983/16645315/a02ac4c6-4455-11e6-9708-04bd3beba518.png)

**Comment by aymericdamien** [comment]
You can try to use a 'validation_set', or you can use 'validation_split' to automatically split your data into a training and validation set, and compare performances.
